### ISSUE TITLE
Behat tests fail out of the box with Pipelines

### ISSUE THREAD

**Issue by danepowell** [acquia/blt#1799]
The out of the box Behat configuration works fine on TravisCI, but fails on Acquia Pipelines using the default Pipelines scripts.

Making these changes to Behat's example.local.yml got Behat running on Pipelines, but broke it on Travis. I'm not sure how to get it working on both out of the box 😖 

````
diff --git a/tests/behat/example.local.yml b/tests/behat/example.local.yml
index 5c7923e..484e871 100644
--- a/tests/behat/example.local.yml
+++ b/tests/behat/example.local.yml
@@ -23,8 +23,10 @@ local:
         - Drupal\DrupalExtension\Context\MessageContext
         - Drupal\DrupalExtension\Context\DrushContext
   extensions:
+    DMore\ChromeExtension\Behat\ServiceContainer\ChromeExtension: ~
     Behat\MinkExtension:
-      javascript_session: selenium2
+      browser_name: chrome
+      javascript_session: default
       # configure the base url for your site
       base_url: ${project.local.uri}
       # set default command for "Show last response" step.
@@ -40,8 +42,12 @@ local:
       #       CURLOPT_CERTINFO: false
       #       CURLOPT_TIMEOUT: 120
       selenium2:
        wd_host: http://127.0.0.1:4444/wd/hub
         browser: chrome
+      sessions:
+        default:
+          chrome:
+            api_url: "http://127.0.0.1:9222"
     Drupal\DrupalExtension:
       drupal:
         # This must be an absolute path.
````

Thanks to @aweingarten for the working config

**Comment by grasmash** [comment]
I'm not able to replicate this issue. The [BLTed project](https://github.com/acquia-pso/blted8)  runs a pipelines build after every commit. This build is passing.  Additionally, the Canary bill this passing.

**Comment by danepowell** [comment]
It looks like maybe this was fixed in BLT 8.9.0. Maybe I upgraded this project from 8.8.x and didn't get the necessary changes.

I'm working on spinning up a new project that might allow me to test this on a fresh 8.9 install, I'll assume it's fixed until then.
